stdlib/LinearAlgebra/src/adjtrans.jl

```diff
@@ -9,10 +9,12 @@ import Base: length, size, axes, IndexStyle, getindex, setindex!, parent, vec, c
 # but also factorizations, rotations, and other linear algebra objects, including
 # user-defined such objects. so do not restrict the wrapped type.
 """
-    Adjoint(A)
+    Adjoint
 
-Constructs an `Adjoint` view of `A`.
-Note that [`adjoint`](@ref) is applied recursively to elements.
+Lazy wrapper type for a adjoint view of the underlying linear algebra object,
+usually an `AbstractVector`/`AbstractMatrix`, but also some `Factorization`, for instance.
```

Comment:
Included `Factorization` as mentioned in the code comments above. If accepted, this would deserve a proper reference once the `Factorization` docstring is written.